default_positions.py

```diff
@@ -0,0 +1,82 @@
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import Session
+
+from __init__ import app
+from connection_initializer import init_tcp_connection_engine
+from connection_initializer import init_unix_connection_engine
+from connection_initializer import load_secrets
+
+Base = declarative_base()
+
+
+class DefaultPosition(Base):
+    """Model of DefaultPositions for SQLAlchemy."""
+
+    __tablename__ = "default_positions"
+    __table_args__ = ({"mysql_charset": "utf8mb4"})
+    id = Column(Integer, autoincrement=True, primary_key=True)
+    organizations_id = Column(Integer, nullable=False)
+    name = Column(String, nullable=False)
+    database_name = "master_data"
+
+
+class DefaultPositionManager:
+    """DefaultPositionManager for Connect Cloud SQL."""
+
+    database_engine = None
+    database_config = {
+        "echo": True,
+        "pool_size": 5,
+        "max_overflow": 2,
+        "pool_timeout": 30,
+        "pool_recycle": 1800
+    }
+    secrets = {"CLOUD_SQL_CONNECTION_NAME": "", "DB_USER": "", "DB_PASSWORD": ""}
+
+    def __init__(self):
+        """Initialize DefaultPositionManager."""
+
+        DefaultPositionManager.secrets = load_secrets(DefaultPositionManager.secrets)
+
+        if app.config["IS_LOCAL"]:
+            self.database_engine = init_tcp_connection_engine(
+                DefaultPositionManager.secrets, DefaultPosition.database_name, DefaultPositionManager.database_config)
+        else:
+            self.database_engine = init_unix_connection_engine(
+                DefaultPositionManager.secrets, DefaultPosition.database_name, DefaultPositionManager.database_config)
+
+        Base.metadata.create_all(bind=self.database_engine)
+
+    def get_default_positions(self, id: int) -> dict:
+        """Get Default Positions.
+
+        Args:
+            id (int): ID of a registered user in organizations.
+
+        Returns:
+            Dict<Default Positions>: Registration Location.
+        """
+
+        session = Session(bind=self.database_engine)
+        default_positions_dictionary = {}
+        default_positions_statement = text(
+            "select id, organizations_id, name from default_positions where id = %s" % (id))
+        default_positions_result = session.execute(default_positions_statement).fetchall()
+        if (default_positions_result == []):
+            app.logger.info("get_default_positions(): The registrants you specify do not exist")
+            return {"id": None, "organizations_id": None, "name": None}
+        else:
+            app.logger.info("get_default_positions(): Successfully obtained default_positions_dictionary")
+            for row in default_positions_result:
+                default_positions_dictionary["id"] = row.id
+                default_positions_dictionary["organizations_id"] = row.organizations_id
+                default_positions_dictionary["name"] = row.name
+            return default_positions_dictionary
```

Comment:
```suggestion
        if default_positions_result:
            app.logger.info("get_default_positions(): Successfully obtained default_positions_dictionary")
            for row in default_positions_result:
                default_positions_dictionary["id"] = row.id
                default_positions_dictionary["organizations_id"] = row.organizations_id
                default_positions_dictionary["name"] = row.name
            return default_positions_dictionary
        else:
            app.logger.info("get_default_positions(): The registrants you specify do not exist")
            return {"id": None, "organizations_id": None, "name": None}
```

Pythonでは空配列は `False` です。
cf. [真理値判定](https://docs.python.org/ja/3/library/stdtypes.html)

Reply:
> Organizationってログインしてからあちこちで使うわけで、あちこちでOrganizationManagerのインスタンスを持つってことになるのかねぇ。なんか、嫌だねぇ。
registrants も default_positions も organization_id さえ知ってれば全部引けるのにねぇ。

そうですね。うーん。
鹿島田さんとも同じ話をしていて、でも他にどう処理するか良いものを思いついていないです...

Reply:
なんで？
ログイン成功してる状態が続く限り、最低でも `organization_id` は持ち続けなきゃいけないんだから、ログイン周りに組み込めば良いじゃん。

Reply:
なるほどですね。
その通りだと思うので、共通化を試みてみます。